encoding/json/encoding_test.go

```diff
@@ -677,15 +677,15 @@ func TestEncodeResource(t *testing.T) {
 		t.Parallel()
 
 		script := `
-			access(all) resource Foo {
-				access(all) let bar: Int
+			pub resource Foo {
```

Comment:
I thought access(all) and pub were equivalent. Why the change? just for clarity?

Reply:
Yeah, all tests use `pub` and I just replaced it for consistency reasons.